.github/workflows/expand-release-detail.yml

```diff
@@ -0,0 +1,37 @@
+name: Add diff to release
+on:
+  release:
+    types: [published]
+
+jobs:
+  show_diff:
+    name: Show ts diff in release notes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+
+      - uses: actions/cache@v2
+        id: node-modules-cache
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+
+      - name: NPM Install
+        run: npm install
+
+      - name: Generate diff
+        run: npm run --silent generate:diff > diff.md
+
+      - name: Edit Release
+        uses: irongut/EditRelease@v1.2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It gives access to pretty much everything, but is only valid for the duration of the job run. This is somewhat unavoidable, since the action needs to modify a GitHub release. However, since it's such a simple thing (basically just calling a github API endpoint), I could just write an action to work in the same way, and avoid the issue

Reply:
@mrbbot I've changed the action to use the GitHub API directly